spec/credit_transfer_spec.rb

```diff
@@ -62,6 +62,12 @@
           expect(subject.to_xml).to validate_against('pain.001.003.03.xsd')
         end
 
+        it 'should fail for pain.001.001.03' do
+          expect {
+            subject.to_xml(SEPA::PAIN_001_001_03)
+          }.to raise_error(RuntimeError)
+        end
```

Comment:
(this adds an unrelated test to cover the mandatory presence of the BIC for the debtor)